js/scripts.js

```diff
@@ -4,4 +4,5 @@ let number = Math.ceil(Math.random() *20);
 console.log(number);
 
 let switchImg = document.getElementById('image').style.backgroundImage = "url('../img/magic*ball_' + number + '.png')";
```

Comment:
you're on the right track, i see a typo in the file name here

Reply:
Well I feel silly. lol. I fixed it in Atom, so I'll push it in just a sec. Still isn't working. I think I'm gonna put a div around the img tag so I can give it a border and see where the image is supposed to show up.

Reply:
I pushed up the new code with the typo fixed